src/jansson.h

```diff
@@ -291,7 +291,14 @@ int json_vunpack_ex(json_t *root, json_error_t *error, size_t flags, const char
 
 /* sprintf */
 
+#ifdef __GNUC__
+__attribute__((format(printf, 1, 2)))
+#endif
```

Comment:
Is this attribute correct before the function declaration? In the examples in https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html, attributes are always specified after the function declataion.

Reply:
Yes. You can use the attributes before and after declaring the function. If you want - I can move the attribute after the function declaration.